src/Nethermind/Nethermind.Network/P2P/Messages/DisconnectMessageSerializer.cs

```diff
@@ -17,26 +17,13 @@ public byte[] Serialize(DisconnectMessage msg)
             ).Bytes;
         }
 
-        private byte[] breach1 = Bytes.FromHexString("0204c104");
-        private byte[] breach2 = Bytes.FromHexString("0204c180");
-
         public DisconnectMessage Deserialize(byte[] msgBytes)
         {
             if (msgBytes.Length == 1)
             {
                 return new DisconnectMessage((DisconnectReason)msgBytes[0]);
             }
 
-            if (msgBytes.SequenceEqual(breach1))
-            {
-                return new DisconnectMessage(DisconnectReason.Breach1);
```

Comment:
Have you checked if it is not used anymore (for example by running node and logging this)?

Reply:
No, and imo, it does not matter. Since it is not part of the spec. You can check the spec at the devp2p repo

Reply:
Well, if I were you I would run the node and try to log it. It will take you two minutes ;) IMO, it doesn't matter too, but we can quickly prove it. However, you can't create assumptions only on spec which might be terribly outdated. If someone added it even if it wasn't in the spec it means that there were reasons for doing it. I read what Tomasz said, but just wondering if we still have such peers.